staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -1067,6 +1069,17 @@ type RunAsUserStrategyOptions struct {
 	Ranges []IDRange `json:"ranges,omitempty" protobuf:"bytes,2,rep,name=ranges"`
 }
 
+// RunAsGroupStrategyOptions defines the strategy type and any options used to create the strategy.
+// Deprecated: use RunAsGroupStrategyOptions from policy API Group instead.
```

Comment:
@liggitt do you know how this works as a multi-group API? Do we need to add it here, or can we just add new fields to the policy group API?

Reply:
yeah, needs to be added here for round trip support